Perl modules are installed in $Config{vendorlib}.
This is the correct location for packaged versions.

Provided by: Alexis Sukrieh <sukria@sukria.net>

---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Makefile
+++ b/Makefile
@@ -30,7 +30,7 @@
 
 # Overwrite that variable with the Perl vendorlib Config value if 
 # you package Backup Manager
-PERL5DIR?="$(DESTDIR)$(shell perl -MConfig -e 'print "$$Config{sitelib}"')"
+PERL5DIR?="$(DESTDIR)$(shell perl -MConfig -e 'print "$$Config{vendorlib}"')"
 
 # Some static paths, specific to backup-manager
 LIBDIR=$(DESTDIR)/usr/share/backup-manager
